packages/web/src/features/entitlements/server.ts

```
@@ -25,29 +44,62 @@ export const getPlan = (): Plan => {
 
     const licenseKey = env.SOURCEBOT_EE_LICENSE_KEY;
     if (licenseKey && licenseKey.startsWith(eeLicenseKeyPrefix)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)